src/main/java/org/example/DatabaseHelper.java

```diff
@@ -0,0 +1,71 @@
+package org.example;
+
+import java.sql.*;
+import java.text.MessageFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+/*
+In this database mySQL database with table name audit.
+The table contains following columns, path to the file, the word searched, the date time of the search
+with error and success messages
+ */
+public class DatabaseHelper {
+    /*
+      Storing data to database
+     */
+    public void storingDataToDataBase(String filepath, String WordSearch, String finalResult, int totalNumberOfWords, String errorMessage) throws SQLException {
```

Comment:
Method name not proper. Java method names should resemble a command.
```suggestion
    public void storeDataToDataBase(String filepath, String WordSearch, String finalResult, int totalNumberOfWords, String errorMessage) throws SQLException {
```

Reply:
Updated.